### ISSUE TITLE
QString multi-arg refactoring for increasing performance.

### ISSUE THREAD

**Issue by dsm** [ra3xdh/qucs_s#868]
Hi, I search code base with \.arg\([^)]*\)\.arg this reg exp and it founds more than 800 lines of code.

Using multi-arg methods gives more performance and less memory usage. Assign this issue to me.

**Comment by ra3xdh** [comment]
Yes, the QtCreator shows warning on `arg().arg().arg()` chains. It is widely used in the project. Such call is not recommended in new Qt versions. The refactoring would improve the code quality.

**Comment by ra3xdh** [comment]
Closed via #1684
